api/testtriggers/v1/testtrigger_types.go

```diff
@@ -238,6 +243,36 @@ type TestTriggerActionParameters struct {
 	Target *commonv1.Target `json:"target,omitempty" expr:"include"`
 }
 
+// TestTrggerContentSelector is used to track content changes
+type TestTrggerContentSelector struct {
```

Comment:
FYI You have a typo here TestTrggerContentSelector -> TestTriggerContentSelector